### PR TITLE
chore: adjusted project structure and added README

### PR DESCRIPTION
**Pull request by machielvdw**

### CODE REVIEW

File: app/lowdefy.yaml [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-  homePageId: home
-  theme:
-    lessVariables:
-      '@primary-color': '#438ef7'
-      '@border-radius-base': 6px
-      '@border-radius-sm': 4px
-      '@border-color-base': '#69c0ff'
-      'border-width-base': 2px
+# config:
+#   homePageId: home
```
Comment (SamTolmay) [inline app/lowdefy.yaml]: Don't we need to define a home page?

File: app/lowdefy.yaml [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+#   theme:
+#     lessVariables:
+#       "@primary-color": "#438ef7"
```
Comment (SamTolmay) [inline app/lowdefy.yaml]: Could you add these in the `public/styles.less` file?

File: app/lowdefy.yaml (hunk 1 above)
Comment (machielvdw) [reply app/lowdefy.yaml]: 80a9093d631a3cc914ed4aaa38e2cb6b4cff2ca4

File: app/lowdefy.yaml (hunk 2 above)
Comment (machielvdw) [reply app/lowdefy.yaml]: 80a9093d631a3cc914ed4aaa38e2cb6b4cff2ca4

**Pull request merged**